test/Twig/Tests/TemplateTest.php

```diff
@@ -262,6 +266,8 @@ public function testGetAttributeStrict($defined, $value, $object, $item, $argume
             } catch (Twig_Error_Runtime $e) {
                 if (null !== $exceptionMessage) {
                     $this->assertSame($exceptionMessage, $e->getMessage());
+                } else {
+                    $this->addToAssertionCount(1);
```

Comment:
Not that easy as we have some tests that throw exceptions without a message. Will see how to fix that later on.